test/static/quantity_test.cpp

```diff
@@ -24,6 +24,7 @@
 #include <mp-units/bits/hacks.h>
 #include <mp-units/ext/fixed_string.h>
 #include <mp-units/ext/type_traits.h>
+#include <mp-units/math.h>
```

Comment:
This header and its features should be used only in a HOSTED (not freestanding build). Please see other similar usages of this header file.